<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tic_tac_toe_ui::SandboxWindow</class>
 <widget class="QFrame" name="tic_tac_toe_ui::SandboxWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>414</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>130</width>
    <height>118</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="tic_tac_toe_ui::BoardWidget" name="board">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>171</width>
       <height>271</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="reset">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>New Game</string>
      </property>
      <property name="icon">
       <iconset resource="../../darwin_studio/resources.qrc">
        <normaloff>:/resources/refresh.png</normaloff>:/resources/refresh.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="flip_sides">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Flip Sides</string>
      </property>
      <property name="icon">
       <iconset resource="../../darwin_studio/resources.qrc">
        <normaloff>:/resources/view_refresh.png</normaloff>:/resources/view_refresh.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="first_move">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>First Move</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../darwin_studio/resources.qrc">
        <normaloff>:/resources/mc_skip_backward.png</normaloff>:/resources/mc_skip_backward.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="last_move">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Last Move</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../darwin_studio/resources.qrc">
        <normaloff>:/resources/mc_skip_forward.png</normaloff>:/resources/mc_skip_forward.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="prev_move">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Previous Move</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../darwin_studio/resources.qrc">
        <normaloff>:/resources/mc_backward.png</normaloff>:/resources/mc_backward.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="next_move">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>210</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Next Move</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../darwin_studio/resources.qrc">
        <normaloff>:/resources/mc_forward.png</normaloff>:/resources/mc_forward.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="x_player_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="o_player_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>O</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tic_tac_toe_ui::BoardWidget</class>
   <extends>QFrame</extends>
   <header>board_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../darwin_studio/resources.qrc"/>
 </resources>
 <connections/>
</ui>
